feat: Add RatingControlSamplePage

lleibovich opened the pull request:
Adds the sample page for the RatingControl.

GitHub Issue (If applicable): #448 

<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal) -->

## PR Type

What kind of change does this PR introduce?
<!-- Please uncomment one ore more that apply to this PR -->

- Feature

## What is the current behavior?

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
No sample page for RatingControl.

## What is the new behavior?

<!-- Please describe the new behavior after your modifications. -->
A sample page for RatingControl has been added.

On Android 12 emulator:
![RatingControl_Android](https://user-images.githubusercontent.com/5546544/197661689-451fe525-a044-4021-9525-62d1f3fa773d.gif)

On Windows 11 PC:
![RatingControl_UWP](https://user-images.githubusercontent.com/5546544/197661676-15ef80bd-63cb-4101-96d0-e689d1286726.png)

On WASM:
![RatingControl_WASM](https://user-images.githubusercontent.com/5546544/197661643-ef4af73a-3567-4fc7-843a-5fddde653673.png)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [x] Tested on Wasm.
- [x] Tested on Android.
- [x] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->


## Other information

<!-- Please provide any additional information if necessary -->

Internal Issue (If applicable):
<!-- Link to relevant internal issue if applicable. All PRs should be associated with an issue (GitHub issue or internal) -->

MartinZikmund commented:
Thanks for the PR! Please move the control under UI Components category (as it is a built-in WinUI control, not part of Toolkit). Very nice work!

lleibovich commented:
> Thanks for the PR! Please move the control under UI Components category (as it is a built-in WinUI control, not part of Toolkit). Very nice work!

Thanks!
I've moved the control and I've also updated the images attached to the PR to reflect this change.
